conversation/python/sdk/conversation/app.py

```diff
@@ -0,0 +1,34 @@
+# ------------------------------------------------------------
+# Copyright 2025 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+from dapr.clients import DaprClient
+from dapr.clients.grpc._request import ConversationInput
+
+with DaprClient() as d:
+    inputs = [
+        ConversationInput(content="What is Dapr?", role='user', scrub_pii=True),
+    ]
+
+    metadata = {
+        'model': 'foo',
```

Comment:
Best not to use the word foo.
```suggestion
        'model': 'modelname',
```